src/rules/tsrDetectPossibleTimingAttacksRule.ts

```diff
@@ -1,19 +1,79 @@
 import * as ts from 'typescript';
 import * as Lint from 'tslint';
 
-const keywordMask = new RegExp('^((' + [
+const keywordMask = new RegExp('^.*((' + [
     'password',
     'secret',
     'api',
     'apiKey',
     'token',
-    'auth',
```

Comment:
why do you think that `auth` keyword shouldn't be a part of this validation?

Reply:
Admittedly it's pretty anecdotal, but in our codebase, any time the `auth` keyword was found was either a false positive or already covered by another keyword (mostly `token`). That said, not super married to that and it can definitely be left in if you think that's best!